codegenerator/cli/npm/envio/src/Types.ts

```diff
@@ -0,0 +1,22 @@
+export type Invalid = never;
+
+export type Address = string;
```

Comment:
I was wondering if at we should use the same type as viem for ts address? Like `0x${string}`

Might be breaking though. But users sometimes have to type cast to turn viem address into string for eg.

Reply:
A good thing to consider. I don't think it's going to be a breaking breaking change